src/cmd/script_args/mod.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+use move_core_types::language_storage::TypeTag;
+use type_args::TypeArgVec;
+
+mod type_args;
+
+/// Arguments for script functions.
+#[derive(Debug, Parser)]
+pub struct ScriptFunctionArguments {
+    /// Type args.
+    #[clap(flatten)]
+    pub(crate) type_arg_vec: TypeArgVec,
```

Comment:
What is the background for this visibility pattern? Just for my personal interest... :blush: Why not `pub`?

Reply:
> What is the background for this visibility pattern?

That's a good question that I'll answer by removing it :D
Since we want to show that this struct element won't be used anywhere else in the codebase, probably ever, and no one should ask the same question again.